watertap/data/techno_economic/component_list.yaml

```
@@ -64,6 +64,8 @@ nitrite:
   constituent_longform: Nitrite
 nitrogen:
   constituent_longform: Nitrogen
+nonbiodegradable_cod:
+  constituent_longform: Nonbiodegradable Chemical Oxygen Demand (ffCOD)
```

Should this (and the `nonvolatile_toc` below) have names without acronyms? E.g.
```suggestion
nonbiodegradable_chemical_oxygen_demand:
  constituent_longform: Nonbiodegradable Chemical Oxygen Demand (ffCOD)
```

---

Generally we have used shorter names as the key, and expanded on them in the `constituent_longform`.

For my part, what does the `ff` in `ffCOD` mean (i.e. just making sure it isn't a typo or issue with importing the csv files)?

---

ffCOD stands for flocculated and filtered chemical oxygen demand. In the case of this particular unit, ffCOD is representative of volatile fatty acids (VFAs), hence the more elaborate longform presented in the unit's yaml file. However, the general longform that isn't unit specific should not describe this component as "representative of volatile fatty acids", which is why I wrote it this way here.

---

As an observation then, we want to be really careful with things like this. What happens if we have two unit operations in a process which use different definitions for the same component (i.e. one unit uses the more elaborate definition, and one uses a more general definition)?